src/Http/Requests/Admin/Request.php

```diff
@@ -93,25 +93,18 @@ private function updateRules($rules, $fields, $locale, $localeActive = true)
                 $fieldRules = explode('|', $fieldRules);
             }
 
-            $fieldRules = Collection::make($fieldRules);
+            $fieldRules = collect($fieldRules);
 
-            // Remove required rules, when locale is not active
+            // Remove required rules and add nullable rules, when locale is not active
             if (! $localeActive) {
-                $hasRequiredRule = $fieldRules->contains(function ($rule) {
-                    return $this->ruleStartsWith($rule, 'required');
-                });
+                $fieldRules = $fieldRules->reject(fn ($rule) => $this->ruleStartsWith($rule, 'required'));
 
-                $fieldRules = $fieldRules->reject(function ($rule) {
-                    return $this->ruleStartsWith($rule, 'required');
-                });
-
-                // @TODO: Can be replaced with doesntContain in twill 3.x
-                if ($hasRequiredRule && !in_array($fieldRules, $fieldRules->toArray())) {
```

Comment:
This is meant for a different PR, but if you're okay with it we can ship it direclty as well, I was just chasing bugs and I stumbled accross
```in_array($fieldRules, $fieldRules->toArray())``` which is very weird and also always true